lib/tasks/data.rake

```diff
@@ -84,6 +84,40 @@ namespace :aleftng do
       "complement" => 'Complement'
   }
 
+  def import_Concepts(concept_names,learning_object)
+    #puts "CONCEPT_NAMES: #{concept_names}"
+    splitted_concept_names = (concept_names.gsub!(",", "\n") || concept_names).split(/\r?\n/)
```

Comment:
`gsub!` (meni povodny objekt) zmenit na `gsub` (nemeni povodny objekt) a vrati to hodnotu aj ked tam vzor nie je a netreba robit tu zbytocnu konstrukciu s `||`.